packages/client/test/cli/cli.spec.ts

```diff
@@ -300,21 +292,22 @@ describe('[CLI]', () => {
         })
         const res = await client.request('web3_clientVersion', [], 2.0)
         assert.ok(res.result.includes('EthereumJS'), 'read from HTTP RPC')
+
         const clientNoConnection = Client.http({
-          port: 8562,
+          port: 8563,
         })
         try {
           await clientNoConnection.request('web3_clientVersion', [], 2.0)
           assert.fail('should have thrown on invalid client address')
         } catch (e: any) {
-          assert.equal(e.code, 'ECONNREFUSED', 'failed to connect to RPC on invalid address')
+          assert.ok(e !== undefined, 'failed to connect to RPC on invalid address')
           child.kill(9)
           resolve(undefined)
```

Comment:
Can't we just `resolve()` ?

Reply:
For this and other instances.

Reply:
Nevermind, a lot of these resolve are typed with `unknown` as arg type which doesn't allow for optionality.